test/src/main/java/org/axonframework/test/fixture/AxonTestPhase.java

```diff
@@ -231,16 +242,36 @@ default Given command(@Nonnull Object payload, @Nonnull Map<String, ?> metaData)
         Given commands(@Nonnull CommandMessage<?>... messages);
 
         /**
-         * Transitions to the "when" phase to execute the test action.
+         * Configures the given {@code commands} as commands in the "given" state. These commands will be dispatched in
+         * the order they are provided in the same Unit of Work.
+         *
+         * @param commands The command messages to dispatch.
+         * @return The current Given instance, for fluent interfacing.
+         */
+        default Given commands(@Nonnull Object... commands) {
```

Comment:
I think the commands should each get their own UoW, or they won't be able to see each other's events right?
Usually this is part of a flow

Reply:
For events it makes sense to have multiple in one UoW, but for commands not.

Reply:
Hmmm... I'm not sure about that. Your assumption is safer.

But... what if you'd like to execute multiple commands transactionally? So in the same unit of work. It was possible in AF4 to execute two commands using same Unit Of Work, if SimpleCommandBus used.